### PR TITLE
Add markdown support

### PR DESCRIPTION
**Pull request by harrisj**

### CODE REVIEW

**Comment by harrisj** [comment]
This probably could use a helper instead of those `<%= raw %>` declarations, but I need someone else to weigh in

File: app/models/presenter/auction.rb [hunk 1]
```diff
@@ -70,6 +75,16 @@ def current_bid_record
       @current_bid_record ||= bids.sort_by{|bid| [bid.amount, bid.created_at, bid.id]}.first
     end
 
+    def markdown
+      # FIXME: Do we want the lax_spacing?
```
Comment (adelevie) [inline app/models/presenter/auction.rb]: what is `lax_spacing`?

**Comment by adelevie** [comment]
screen shot 2015-12-23 at 4 42 52 pm

Seems to be missing any `h` tags.

**Comment by adelevie** [comment]
screen shot 2015-12-23 at 5 47 19 pm

**Comment by adelevie** [comment]
Did some rebasing.

**Comment by harrisj** [comment]
We should add shared specs to eliminate the duplication in the markdown tests, but I am not sure how to do that exactly yet and I want to get this in there.

**Pull request merged**